src/main/kotlin/util/SphereNodes.kt

```diff
@@ -108,21 +109,23 @@ class SphereNodes //FAILED to hook up here a 'parentNode' listener that would se
         val g = (intColor shr 8 and 0x000000FF) / 255f
         val b = (intColor and 0x000000FF) / 255f
         node.material().diffuse[r, g] = b
+//        node.material().diffuse = Vector3f(r, g, b)
+//        node.material().blending = Blending(transparent = true, opacity = 0.2f)
         node.material().wireframe = false
     }
 
     fun decreaseSphereScale() {
         val oldScale = SCALE_FACTOR
```

Comment:
this is IMHO important that this got preserved -- the independent scaling of spheres w.r.t. volume
(I was worried when I saw above removal of `spotRadiusScale`)